src/components/Pages/AboutUsPage/AboutUsPage.js

```diff
@@ -62,8 +63,16 @@ class AboutUsPage extends React.Component {
     const donateMessage =
       donatePageData && donatePageData.title ? donatePageData.title : null;
 
+      const {community} = this.props;
+
     return (
       <div className="boxed_wrapper">
+        {Seo({
+          title: "About Us",
```

Comment:
Perhaps all titles could be: "{CommunityName} - About Us"

Reply:
the title will appear in the browser and GA as `subdomain | title`